### DISCUSSION TITLE
Ajuda com erro durante a instalação

### DISCUSSION THREAD

**Discussion by DMFSouza** [po-ui/po-angular#1298] [Angular]
Gente estou tentando iniciar no mundo do PO-UI, inclusive não sei se este é o lugar mais aconselhável para postar esta dúvida, caso não seja desde já peço desculpas. Segui os passos da pagina oficial "primeiros passos", mas na hora de correr o comando "ng add @po-ui/ng-components" retorna este erro... já verifiquei o arquivo package.json e aparentemente esta tudo certo, já tentei reinstalar diversas vezes e sempre ocorre o mesmo erro... Poderiam me ajudar por favor?
![image](https://user-images.githubusercontent.com/92698142/172064359-edf6c652-92ed-4b4e-8580-79a3d77cee7a.png)
![image](https://user-images.githubusercontent.com/92698142/172064375-a17d19b0-4191-40b7-a4ed-08db8fed513a.png)

**Comment by rafaellmarques** [comment]
Bom dia @DMFSouza , tudo bem?
Analisei as imagens que disponibilizou e vi que a versão do `rxjs` não está de acordo com a versão que estamos usando no PO-UI.
Tente refazer os primeiros passos e antes de adicionar os componentes no seu projeto altere a versão do `rxjs` no arquivo `package.json`.

`rxjs: "~7.4.0"`
